.github/workflows/image.yaml

```diff
@@ -0,0 +1,179 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run this workflow on pull requests or merge to main/release branches
+name: image
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+    branches:
+      - main
+      - release-*
+  push:
+    branches:
+      - main
+      - release-*
+
+jobs:
+  image:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        driver:
+          - 470.223.02
```

Comment:
Note, 470 is not supported on ubuntu22.04 and rhel9, so we need to skip this version for those distros. See https://gitlab.com/nvidia/container-images/driver/-/blob/main/.common-ci.yml?ref_type=heads#L78 and https://gitlab.com/nvidia/container-images/driver/-/blob/main/.common-ci.yml?ref_type=heads#L91